protoc-gen-swagger/genswagger/template.go

```diff
@@ -84,6 +84,10 @@ var wktSchemas = map[string]schemaCore{
 	},
 }
 
+var regBool = `^true|false$`
+
+var regString = `^"\w+"$`
```

Comment:
This is equivalent to ^"[a-zA-Z0-9_]+"$ which seems needlessly restrictive. As far as I can tell, a string is defined as "a sequence of zero or more Unicode characters" (ref: https://tools.ietf.org/html/rfc4627, referenced from https://tools.ietf.org/html/draft-zyp-json-schema-04#section-2, referenced from https://swagger.io/specification/v2/#data-types). With that background, I don't think we need a definition of a string at all, the protobuf parser will sort out whether something is a sequence of zero or more unicode characters, as long as it has reached us here in once piece, it's a valid string. We could require that the first and last character in the string is a quote, that seems fine.